forc/src/utils/defaults.rs

```diff
@@ -53,7 +53,7 @@ pub(crate) fn default_program() -> String {
     r#"script;
 
 fn main() {
-    
+
 }
```

Comment:
Omg I did it again https://github.com/FuelLabs/sway/pull/766#discussion_r803492506

I'm honestly not intending to do this, my editor is just accidentally "fixing" this and I keep missing it lol (I'm gonna leave it unless someone requests otherwise)